src/main/auth/service.ts

```diff
@@ -1,19 +1,52 @@
+import { safeStorage } from 'electron';
+import Logger from 'electron-log';
 import ConfigStore, { defaults, fieldsToSave } from '../config';
 import packageConfig from '../../../package.json';
 import { User } from '../types';
 
+const TOKEN_ENCODING = 'latin1';
```

Comment:
does latin1 support every character a JWT can have?

Reply:
The `latin1` is to encode only the buffer resultant of the encryption. when it's decrypted the buffer it's created with the same encoding so it should be ok